pkg/plugins/manager/signature/manifest.go

```diff
@@ -194,6 +194,9 @@ func Calculate(ctx context.Context, mlog log.Logger, src plugins.PluginSource, p
 	// Track files missing from the manifest
 	var unsignedFiles []string
 	for _, f := range plugin.FS.Files() {
+		// Ensure slashes are used, because MANIFEST.txt always uses slashes regardless of the filesystem
+		f = strings.ReplaceAll(f, "\\", "/")
```

Comment:
I used this rather than `filepath.ToSlash` because the implementation changes depending on the running OS (`Separator` is different on Linux/Windows), and we cannot use it to test cross-platform.

stdlib source:

```go
// ToSlash returns the result of replacing each separator character
// in path with a slash ('/') character. Multiple separators are
// replaced by multiple slashes.
func ToSlash(path string) string {
	if Separator == '/' {
		return path
	}
	return strings.ReplaceAll(path, string(Separator), "/")
}
```

Reply:
I prefer `filepath.ToSlash` because it leaves untouched the string if running on linux (imagine a corner case of a file with a path of `a/b\c` in Linux :sweat_smile:). You can define `toSlash` as a package variable and replace it in the test to do what you want:

```go
var (
	toSlash = filepath.ToSlash
)
```